app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -35,7 +35,7 @@ def show
     param :product_id, :number, :required => true, :desc => N_("ID of a product to list repository sets from")
     def available_repositories
       scan_cdn = sync_task(::Actions::Katello::RepositorySet::ScanCdn, @product, @product_content.content.id)
-      repos = scan_cdn.output[:results]
+      repos = scan_cdn.output[:results] || []
```

Comment:
It happens when the CDN returns access forbidden (i.e. `CDN loading error: access forbidden to https://cdn.redhat.com:443/content/dist/rhel/server/7/7Server/listing (Katello::Errors::SecurityViolation)`). I believe this is caused by an outdated or invalid manifest. The workaround on the BZ says the issue was resolved by re-uploading their manifest.